test/test_mediafile.py

```diff
@@ -318,29 +340,30 @@ class ReadWriteTestBase(ArtTestMixin, GenreListTestMixin,
     """
 
     full_initial_tags = {
-        'title':       u'full',
-        'artist':      u'the artist',
-        'album':       u'the album',
-        'genre':       u'the genre',
-        'composer':    u'the composer',
-        'grouping':    u'the grouping',
-        'year':        2001,
-        'month':       None,
-        'day':         None,
-        'date':        datetime.date(2001, 1, 1),
-        'track':       2,
-        'tracktotal':  3,
-        'disc':        4,
-        'disctotal':   5,
-        'lyrics':      u'the lyrics',
-        'comments':    u'the comments',
-        'bpm':         6,
-        'comp':        True,
-        'mb_trackid':  '8b882575-08a5-4452-a7a7-cbb8a1531f9e',
-        'mb_albumid':  '9e873859-8aa4-4790-b985-5a953e8ef628',
-        'mb_artistid': '7cf0ea9d-86b9-4dad-ba9e-2355a64899ea',
-        'art':         None,
-        'label':       u'the label',
+        'title':         u'full',
+        'artist':        u'the artist',
+        'album':         u'the album',
+        'genre':         u'the genre',
+        'composer':      u'the composer',
+        'composer_sort': u'the sortname of the composer',
```

Comment:
true, but I wonder why there isn't artist_sort . Imo where there is a name there also should be the sort name.